modules/ROOT/pages/procedures.adoc

```diff
@@ -826,6 +826,11 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/current/p
 | *Mode* 3+| DBMS
```

Comment:
```suggestion
| `Cypher` | `LIST<STRING>` | The supported  Cypher version.
| *Mode* 3+| DBMS
```

Reply:
But it isnt a new column - it is a row in the `name` column?

Reply:
but the ones above are also rows, no?
```
| `versions` | `LIST<STRING>` | The installed versions of the component.
| `edition` | `STRING` | The Neo4j edition of the DBMS.
```